src/components/Description/Paragraph/index.tsx

```diff
@@ -20,11 +20,13 @@ const CITATIONS_REGEX = `(\\[(${CITATION_REGEX})+\\])`;
 const REMOVE_TRAILING_COMMA_REGEX = /,\s*(?=\])/;
 
 const xReferenceURL = {
-  cazy: 'http://www.cazy.org/fam/{}.html',
-  cog: 'https://ftp.ncbi.nih.gov/pub/COG/COG2014/static/byCOG/{}.html',
-  intenz: 'http://www.ebi.ac.uk/intenz/query?cmd=SearchEC&ec={}',
-  genprop: 'https://www.ebi.ac.uk/interpro/genomeproperties/#{}',
+  cazy: 'http://www.cazy.org/{}.html',
+  cog: 'https://ftp.ncbi.nih.gov/pub/COG/COG2020/data/',
+  intenz: 'https://enzyme.expasy.org/EC/{}',
+  genprop: 'https://www.ebi.ac.uk/interpro/genomeproperties/genome-property/{}',
   superfamily: 'http://supfam.org/SUPERFAMILY/cgi-bin/scop.cgi?ipid={}',
+  enzyme: 'https://enzyme.expasy.org/EC/{}',
```

Comment:
There aren't `[enzyme:...]` cross-references in abstracts (cross-references to EC numbers are created with `[ec:...]`)

```suggestion
```